src/components/RequesterPaysModal.js

```diff
@@ -12,27 +12,28 @@ import { requesterPaysProjectStore } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 
 
-const billingHelpInfo = div({ style: { paddingTop: '1rem' } }, [
+const requesterPaysHelpInfo = div({ style: { paddingTop: '1rem' } }, [
   h(Link, {
     href: 'https://support.terra.bio/hc/en-us/articles/360029801491',
     ...Utils.newTabLinkProps
-  }, ['Why is billing required to access this data?', icon('pop-out', { style: { marginLeft: '0.25rem' }, size: 12 })])
+  }, ['Why is a workspace required to access this data?', icon('pop-out', { style: { marginLeft: '0.25rem' }, size: 12 })])
```

Comment:
Is this article still going to be relevant?

Reply:
I let Allie know to change it.